### PR TITLE
Survey form setup

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this project we:

-Created a form
-Used two types of form input
-Used button radio
-Used dropdown button

### CODE REVIEW

File: surveyform.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<h1  id='title'> title  </title></h1>
```
Comment (Bleu-Yves-Sopoude) [inline surveyform.html:1]: Please give a meaningful name to your form, it's make your page more professional.

File: surveyform.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<p id='description'> Description of the form<p>
+
+<form id="survey-form" >
+  
+    <input type="text" placeholder="Bleu Yves" name="model"><br /><br />
+  
+    <input id="email" placeholder="bleuyves@gmail.com" name="age"><br /><br />
+
+<input type="number" id="quantity" name="quantity" min="1" max="5">
+ <br /><br />
```
Comment (Bleu-Yves-Sopoude) [inline surveyform.html:11]: please stop doubling br tag in a same place it's an orphan tag, only one works

File: surveyform.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+<input type="radio" id="html" name="fav_language" value="HTML">
+<label for="html">HTML</label><br>
+
+<input type="radio" id="html" name="fav_language" value="">
+<label for="html">CSS</label><br>
+
+<textarea id="w3review" name="w3review" rows="4" cols="50">
+Text area
+</textarea>
```
Comment (Bleu-Yves-Sopoude) [inline surveyform.html:27]: Please indent your code, it will be clear and simple to understand when the lines will grow up.

Comment (Bleu-Yves-Sopoude) [inline surveyform.html:27]: Please use linter to check your code before submitting for review, it will help you a lot for a review.

**Review by Bleu-Yves-Sopoude — COMMENTED** [review]
Changed required ♻️ 

Please check my comments below and make all required changes:

Submit for re-review when you are done.

Thanks and happy coding !

File: surveyform.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<h1  id='title'> title  </title></h1>
+<p id='description'> Description of the form<p>
+
```
Comment (joel-kalema) [inline surveyform.html:3]: #### Good job adding the comment 😃, however, there is some point you need to change

- [ ] It should be better to `highlight` the effort of the student to encourage him 💪.  Kindly, add it 😃  
- [ ] Kindly, add some `emojis`  to make your comment more friendly 😃

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Bleu-Yves-Sopoude 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Your comment is polite :heavy_check_mark: 
- You explained `why` :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bleu-Yves-Sopoude — COMMENTED** [review]
**Almost Done** 🛩️ 

Great 💪 You made many efforts and managed several parts well but there are still some elements to improve. 🕹️ 

**To highlight:**

- Working code 💻 
- Code simply to understand ☺️ 


Changed required ♻️

Please check my comments below and make all required changes:

Submit for re-review when you are done.

Thanks and happy coding ! 🍾

**Review by Ibizugbe — APPROVED** [review]
Hi @Bleu-Yves-Sopoude 👋👋👋, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Github `review changes` feature have been used👍.
- [x] Comments are polite and suggestive👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**